### ISSUE TITLE
GIVpower key messaging brainstorm!

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#616]
Need some key messages for:

https://docs.google.com/document/d/1NpyiETNClSw2BGLqE4ucr2eem5VtmZcIpHzFneC0ciE/edit#

**Comment by laurenluz** [comment]
We need key messages from this spec... things/ways we want to communicate this stuff to our users... Some base level stuff:

- GIVpower lets users decide which Giveth projects rise to the top.
- Use GIVpower to help great projects on Giveth shine.
- Stake & lock GIV to enable your GIVpower.
- Boost your favourite projects to help them rise through the ranks.
- Earn a yield on your staked GIV! The longer you lock, the longer you earn.
- New donors to top projects get more GIVbacks.
- Top projects benefit from additional matching funds from the Giveth Matching Pool.

Basically this issue is working on a comms strategy for explaining GIVpower... what are the best ways to communicate this? What are the key phrases we should use on the DApp? On Twitter? 

Would love some more ideas! @danelsuga @WhyldWanderer @divine-comedian @ahmadabugosh

**Comment by danelsuga** [comment]
👍🏼

**Comment by divine-comedian** [comment]
> Top projects benefit from additional matching funds from the Giveth Matching Pool.

Sounds great but I don't think we have finalized this part. 

I'll see if I can think of somethings and add them to the doc.

**Comment by laurenluz** [comment]
@todd-x-y for storytelling as per comms call today
